File: app/src/main/java/com/kksionek/queuedroid/view/keyboard/ButtonViewHolder.java
package com.kksionek.queuedroid.view.keyboard;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.Button;

public class ButtonViewHolder extends RecyclerView.ViewHolder {

    public final Button button;

    public ButtonViewHolder(View itemView) {
        super(itemView);
        button = (Button) itemView;
    }
}

File: app/src/main/java/com/kksionek/queuedroid/view/keyboard/ButtonViewHolder.kt
package com.kksionek.queuedroid.view.keyboard

import androidx.recyclerview.widget.RecyclerView
import com.kksionek.queuedroid.databinding.KeyboardButtonBinding

class ButtonViewHolder(
    private val binding: KeyboardButtonBinding
) : RecyclerView.ViewHolder(binding.root) {
    val button get() = binding.root
}